sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java

```diff
@@ -75,7 +75,7 @@ public class Event extends PseudonymizableAdo {
 	public static final String EVENT_INVESTIGATION_STATUS = "eventInvestigationStatus";
 	public static final String EVENT_INVESTIGATION_START_DATE = "eventInvestigationStartDate";
 	public static final String EVENT_INVESTIGATION_END_DATE = "eventInvestigationEndDate";
-	public static final String EVENT_PERSONS = "eventPersons";
+	public static final String EVENT_PARTICIPANTS = "eventParticipants";
```

Comment:
This is now inconsistent with the name of the field in the same class; you have to rename that to eventParticipants as well

Reply:
In this Event.java class ( from sormas-app/app/src/main/java/de/symeda/sormas/app/backend/event/Event.java) there is no eventPerson or eventParticipant field. 

In Event.java from sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java we had an eventPersons which I've renamed to eventParticipants or are you reffering to something else? :) 

![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/8de7ea16-1ce7-470a-8987-58040aeff06a)